src/modules/directory.rs

```diff
@@ -573,7 +581,7 @@ mod tests {
                 home_symbol = "🚀"
             })
             .collect();
-        let expected = Some(format!("{} ", Color::Cyan.bold().paint("🚀")));
+        let expected = Some(from_slash(&format!("{} ", Color::Cyan.bold().paint("🚀"))));
```

Comment:
I hate to ask this, but could you move the calls to `from_slash` into the tightest scope possible? In this case, it would be something like 

```suggestion
  let expected = Some(&format!("{} ", Color::Cyan.bold().paint(from_slash("🚀"))));
```

Where the call currently is, it's not obvious from context whether the from_slash call requires a formatted color string, or can work on bare paths. We can add a test that tests this in the current scope if you'd like (after coloring is added), but I think the current tests don't accurately reflect how the function is meant to be used.